src/chrome/utils/failures.ts

```diff
@@ -1,3 +1,5 @@
+import { telemetry } from '../../telemetry';
```

Comment:
This should go below the copyright (autoimport doesn't know that...)

Reply:
Done